dash_docs/tutorial/core_components.py

```diff
@@ -62,11 +62,11 @@
 )''', style=styles.code_container),
     html.Br(),
     dcc.Link(html.A('More Dropdown Examples and Reference'),
-             href="/dash-core-components/dropdown"),
+             href="/Docs/dash-core-components/dropdown"),
```

Comment:
```suggestion
             href=tools.relpath("/dash-core-components/dropdown")),
```
Why do we have a `html.A` inside a `dcc.Link`?

(same throughout this page)

Reply:
I think it was back before https://github.com/plotly/dash-core-components/issues/129 was fixed